diffpy/pdfmorph/pdfmorph_api.py

```diff
@@ -0,0 +1,200 @@
+#
+# diffpy.pdfmorph   by DANSE Diffraction group
+#                   Simon J. L. Billinge
+#                   (c) 2018 Trustees of the Columbia University
+#                   in the City of New York.  All rights reserved.
+#
+# File coded by:    Timothy Liu
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+
+from collections import Iterable
+from diffpy.pdfmorph import morphs
+from diffpy.pdfmorph import refine as ref
+from diffpy.pdfmorph import tools
+
+import matplotlib.pyplot as plt
+
+default_config = dict(scale=None, stretch=None, smear=None,
+                      baselineslope=None, qdamp=None)
+
+# map of operation dict
+# TODO: include morphing on psize
+morph_step_dict = dict(scale=morphs.MorphScale,
+                       stretch=morphs.MorphStretch,
+                       smear=[morphs.MorphXtalPDFtoRDF,
+                              morphs.MorphSmear,
+                              morphs.MorphXtalRDFtoPDF],
+                       qdamp=morphs.MorphResolutionDamping)
+
+def pdfmorph(xobj, yobj, xref, yref, rmin=None, rmax=None, rstep=None,
+             pearson=False, addpearson=False, fixed_operations=None,
+             refine=True, verbose=True, **kwargs):
+    """function to perfom PDF morphing.
+
+    Parameters
+    ----------
+    xobj : numpy.array
+        An array of objective x values, i.e., those will be manipulated by
+        morphing.
+    yobj : numpy.array
+        An array of objective y values, i.e., those will be manipulated by
+        morphing.
+    xref : numpy.array
+        An array of reference x values, i.e., those will be kept constant by
+        morphing.
+    yobj : numpy.array
+        An array of reference y values, i.e., those will be kept constant by
+        morphing.
+    rmin : float, optional
+        A value to specify lower r-limit of morph operations.
+    rmax : float, optional
+        A value to specify upper r-limit of morph operations.
+    rstep : float, optional
+        A value to specify rstep of morph operations.
+    pearson: Bool, optional
+        Option to include Pearson coefficient as a minimizing target 
+        during morphing. Default to False.
+    addpearson: Bool, optional
+        Option to include **both** Pearson coefficient and Rw as
+        minimizing targets during morphing. Default to False.
+    fixed_operations : list, optional
+        A list of string specifying operations will be keep fixed during
+        morphing. Default is None.
+    refine : bool, optional
+        Option to execute the minimization step in morphing. If False,
+        the morphing will be applied with parameter values specified in
+        `morph_config`. Default to True.
+    verbose : bool, optional
+        Option to print full result after morph. Default to True.
```

Comment:
can we change the default to `verbose=False`?

Reply:
yes, default verbose should always be false.